mod/layout/set/abstract/media.rb

```diff
@@ -4,7 +4,8 @@ def image_card
   end
 
   def image_src opts
-    image_card&.format&.render_source(size: opts[:size])
+    return "" unless image_card
+    nest image_card, view: :source, size: opts[:size]
```

Comment:
There is a potential problem I was just dealing with elsewhere (and maybe related to your question yesterday).

If I do this:
```
attribute_value = nest cardname, viewname
wrap_with :div, attribute: attribute_value...
```
...then there is the potential for the nest to return a stub and the stub to get escaped as it's put inside the attribute.

possibilities:
1. find a way to avoid the escaping (haml, maybe?)
2. create the view on the image
3. render the image source without a subformat (eg image_card.format.render...).  This will work, but the cache won't get cleared if the image is changed

Reply:
in other words, the old code may be safer :(